docs/user_guides/debugging/tools.rst

```diff
@@ -7,32 +7,31 @@ Consider the following two examples. ``numba_dppy/examples/debug/simple_sum.py``
     :lines: 15-
     :linenos:
 
-Example with njit:
+Example with njit (Numba CPU):
```

Comment:
What do we mean be Numba CPU? Default Numba execution on a CPU?

Reply:
I think yes, Default Numba execution on a CPU

Reply:
Ok. Can you then reword it then?